src/ducks/Signals/SignalNotification.module.scss

```diff
@@ -0,0 +1,4 @@
+
+.notificationStyle {
+  width: 470px;
```

Comment:
In my opinion, that's breaks the notification consistency. @haritonasty @Partysun What do you think?
@GarageInc What is exact reason for changing the width?